src/visualizations/store/adapters/dhis_highcharts/yearOnYear.js

```diff
@@ -1,13 +1,24 @@
-export default function (acc, series, categories, idValueMap) {
-    const serieData = []
+export default function (
+    acc,
+    series,
+    categories,
+    idValueMap,
+    metaData,
+    extraOptions
+) {
+    const serieData = Array.from(
```

Comment:
Even though you didn't create this variable name, you could fix it to `seriesData`

Reply:
Renamed.